units/systemd-tmpfiles-setup.service

```diff
@@ -8,7 +8,7 @@
 #  (at your option) any later version.
 
 [Unit]
-Description=Create Volatile Files and Directories
+Description=Create System-wide Files and Directories
```

Comment:
"System-wide" is not needed, I think. Why not just "Create Files and Directories"?

Reply:
Hmm, I want something more descriptive than that. Any suggestions?

Reply:
"Create Configured Files …"?

Reply:
I dropped "-wide". I think "System Files and Directories" implies that the files are owned by the system (as opposed to users).

"Configured Files" sounds a bit weird to me?

Reply:
I think "create and remove files and directories" would describe the behaviour of this tool accurately. 

After all, it can even remove/purge user home directory with all the files and directories that home directory typically contains.

You have to limit the behaviour first if you think the description should be more limited.